content/en/project/cognitive_dispersion_fMRI/index.md

```diff
@@ -0,0 +1,108 @@
+---
+type: "project" # DON'T TOUCH THIS ! :)
+date: "2025-06-14" # Date you first upload your project.
+title: "2025 Brainhack School (Mini-)Project - Cognitive Dispersion and Its Neural Correlates"
+names: [Truc (Curt) Nguyen]
+github_repo: https://github.com/curt-1711/Curt_brainhack_project
+# If you are working on a project that has website, indicate the full url including "https://" below or leave it empty.
+website:
+# List +- 4 keywords that best describe your project within []. Note that the project summary also involves a number of key words. Those are listed on top of the [github repository](https://github.com/PSY6983-2021/project_template), click `manage topics`.
+tags: [neuropsychology, fMRI, cognitive dispersion, default mode network]
+summary: "This mini-project for the 2025 Brainhack School is part of my PhD dissertation on Late-Life Cognitive Heterogeneity, where I examine the neural correlates of cognitive dispersion -- a measure of within-individual variability -- using neuropsychological and fMRI data from the Midnight Scan Club dataset (OpenNeuro ds000224)."
+#image: ""
```

Comment:
please upload an image to serve as cover of your project. Uncomment this tag, and specify the name of the file here